### PR TITLE
Improve sentences

### PR DESCRIPTION
**Pull request by loic-sharma**

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-5/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1181) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811251219243652-1181%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=VuvTjvMvDDB3mi%2BE3Jbirodlg1whGnDAFII57lFZMiU%3D&st=2018-11-25T12%3A15%3A26Z&se=2018-12-26T12%3A20%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Symbol-Packages-snupkg.md [hunk 1]
```diff
@@ -61,13 +61,13 @@ SymbolsPackageFormat property can have one of the two values: `symbols.nupkg` (t
...
 
-1. You can also push both primary and symbol packages at the same time using the below command. Both, .nupkg and .snupkg files need to be present in the current folder.
+1. You can also push both primary and symbol packages at the same time using the below command. Both .nupkg and .snupkg files need to be present in the current folder.
 
     ```cli
     nuget push MyPackage.nupkg
     ```
 
-In this case, NuGet will publish to nuget.org the `MyPackage.nupkg` first followed by `MyPackage.snupkg`.
+NuGet will publish to nuget.org the `MyPackage.nupkg` first, followed by `MyPackage.snupkg`.
```
Comment (nkolev92) [inline docs/create-packages/Symbol-Packages-snupkg.md]: Doesn't this read a bit weird? 
NuGet will publish the `MyPackage.nupkg` to nuget.org first, followed by `MyPackage.snupkg`.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-5/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1181) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C26%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811261656503261-1181%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LMpWZFf8iHlFhGL8i08mmJVy9Kfasvktx3Im0cbNGcw%3D&st=2018-11-26T16%3A52%3A48Z&se=2018-12-27T16%3A57%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Symbol-Packages-snupkg.md (hunk 1 above)
Comment (loic-sharma) [reply docs/create-packages/Symbol-Packages-snupkg.md]: Yes, you're totally right. What do you think of:

> NuGet will publish both packages to nuget.org. `MyPackage.nupkg` will be published first, followed by `MyPackage.snupkg`.

Comment (nkolev92) [reply docs/create-packages/Symbol-Packages-snupkg.md]: I like it.

**Pull request merged**